cfy_manager/components/postgresql_server/scripts/postgresql_server_wrapper_script.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+function stop_postgres() {
+  echo "Stopping postgres.."
+  echo "Stopping postgres.." >> /tmp/test.txt
```

Comment:
Can we maybe put this output somewhere other than /tmp/test.txt? (probably somewhere in /var/log/cloudify?)

Reply:
this was while i was testing, i thought i removed it